### ISSUE TITLE
Juicer Tools Pre fails to generate KR normalization for a few chromosomes at certain resolutions

### ISSUE THREAD

**Issue by tanlongzhi** [theaidenlab/juicer#176]
Hi,

I'm a long-time enthusiast of Juicer Tools Pre, and have loved it. However, it failed to generate KR normalization for **a few selected chromosome/resolution combinations** for one of my critical .hic files.

In particular, chr13 of my generated .hic file has no KR normalization at 5 kb resolution; all other chromosomes have KR, and chr13 still has other normalization such as VC.

The missing KR normalization is causing a major problem for me, because it prohibits downstream HiCCUP for this one sample only.

Would you mind helping me fix this? Thank you very much in advance!

FYI I'm attaching the verbose log of my command, in case it's helpful for your diagnosis: [pre_verbose_log.txt](https://github.com/aidenlab/juicer/files/5106067/pre_verbose_log.txt)

Best,
Tan

**Comment by sa501428** [comment]
Hey @tanlongzhi !
Thanks for the kind words! Have you tried version 1.22.01? Specifically, there's a developmental type of normalization (SCALE) which may converge when KR fails. You can then pass this into HiCCUPS using -k SCALE. Running AddNorm should suffice (shouldn't need to rerun all of Pre). Let us know how it goes! You can also post questions to the forum: aidenlab.org/forum.html

**Comment by tanlongzhi** [comment]
Hey Muhammad! It's wonderful to hear from you again; and thanks a lot for the prompt help! I suspected failure of KR convergence, but wasn't sure because no error/warning messages appeared during Pre.

My version was a little earlier than 1.22.01; but I did notice that all my files already have SCALE normalization in them -- I just had no idea what SCALE meant. I can now run HiCCUPs with your suggested "-k SCALE" -- really appreciated it!

Best,
Tan
